source/MRMesh/MRVoxelsSave.cpp

```diff
@@ -69,6 +73,8 @@ tl::expected<void, std::string> saveRAW( const std::filesystem::path& path, cons
         ss << "Cannot write file: " << utf8string( outPath ) << std::endl;
         return tl::make_unexpected( ss.str() );
     }
+    if ( !callback( 0.03f ) )
+        return tl::make_unexpected( std::string( "Saving canceled" ) );
```

Comment:
please add callbacks also for actual saving

Reply:
reworked